conf/application.conf

```diff
@@ -152,6 +152,11 @@ vat-agent-client-lookup-frontend {
   path = "/vat-through-software/agent-lookup/client-vat-number"
 }
 
+manage-vat-subscription-frontend {
+  url = "http://localhost:9150"
+  path = "/vat-through-software/account/change-business-details/non-agent"
```

Comment:
An app-config-base PR should be raised whenever you add config, but in this case I've actually already raised one to add config for `manage-vat-subscription-frontend` as part of another task I started. I've just updated the names of the config values in my PR to match yours. Could you have a look at that PR so the changes can be reflected in all environments?

Reply:
Yes I'll have a look at yours now GH is back. Did try to yesterday.